config/co2eq_parameters.test.js

```diff
@@ -5,6 +5,8 @@ describe('Each fallbackZoneMixes sums to one', () => {
     ['defaults', defaultCO2Params.fallbackZoneMixes.defaults],
     ...Object.entries(defaultCO2Params.fallbackZoneMixes.zoneOverrides),
   ])('for zone %s', (zone, fallbackZoneMix) => {
+    // Verify the key exists
+    expect(fallbackZoneMix.powerOriginRatios).toBeTruthy();
```

Comment:
Moved the python test to JS by adding this line.
I did this as the python test was part of the "parser" tests, which didn't make sense